tests/conftest.py

```diff
@@ -144,7 +160,7 @@ def example_figure() -> FigureResampler:
                 name=f"room {i+1}",
             ),
             hf_x=df_data_pc.index,
-            hf_y=df_data_pc[c],
+            hf_y=df_data_pc[c].astype(np.float32),
```

Comment:
why this casting?

Reply:
serialization (smaller size, faster loading / saving)